packages/twenty-front/src/modules/object-record/record-board/record-board-column/components/RecordBoardColumn.tsx

```diff
@@ -14,7 +14,6 @@ const StyledColumn = styled.div<{ isFirstColumn: boolean }>`
       isFirstColumn ? 'none' : theme.border.color.light};
   display: flex;
   flex-direction: column;
-  height: fit-content;
```

Comment:
this was a fix for column higher than the window but this is breaking the case where column height < window height